package.json

```diff
@@ -23,19 +23,18 @@
   },
   "homepage": "https://github.com/fossamagna/babel-preset-gas#readme",
   "dependencies": {
-    "@babel/helper-plugin-utils": "^7.0.0-beta.55",
-    "@babel/preset-env": "^7.0.0-beta.55",
-    "@babel/plugin-transform-member-expression-literals": "^7.0.0-beta.55",
-    "@babel/plugin-transform-property-literals": "^7.0.0-beta.55"
+    "@babel/helper-plugin-utils": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/plugin-transform-member-expression-literals": "^7.0.0",
+    "@babel/plugin-transform-property-literals": "^7.0.0"
   },
   "peerDependencies": {
-    "@babel/core": "7.0.0-beta.55"
+    "@babel/core": "@7.0.0"
```

Comment:
I think it might be caret `^`, not be at sign `@`?

Reply:
Oh, typo there 😅